### PR TITLE
Default to Windows Server version 1809 for Windows nodes.

### PR DESCRIPTION
**Pull request by pjh**
/kind cleanup

**What this PR does / why we need it**:
Changes the default version for Windows nodes to Windows Server version 1809.

Removes all references to 1803, including moving "win1803" directory to just "windows". A single Windows directory suffices for now, if necessary in the future we can shard it into directories for each
Windows version.

We've been running tests with Windows 1809 nodes for a couple days in our fork without major problems: https://testgrid.k8s.io/google-windows#windows-prototype&width=20. Testing on Azure is already using 1809: https://testgrid.k8s.io/sig-windows#Conformance%20acs-engine%20on%20Azure&width=20.

```release-note
NONE
```

### CODE REVIEW

File: cluster/gce/windows/k8s-node-setup.psm1
Comment (yujuhong) [inline cluster/gce/windows/k8s-node-setup.psm1]: Does that mean we'll have to use 1809 nanoserver containers on 2019 hosts?

**Review by yujuhong — COMMENTED** [review]
/lgtm

**Comment by yujuhong** [comment]
/approve
/hold

I'd like to see a few green runs for 1803 before bumping the version.

File: cluster/gce/windows/k8s-node-setup.psm1
Comment (michmike) [reply cluster/gce/windows/k8s-node-setup.psm1]: yes @yujuhong  that's one of the options. you can use either of the 3 following container images on Server 2019
1. servercore:1809
2. nanoserver:1809
3. servercore:ltsc2019

**Review by michmike — APPROVED** [review]

**Comment by michmike** [comment]
/lgtm
Once Yu-Ju removes the hold, we will proceed (let;s see a few green run as mentioned)

**Comment by yujuhong** [comment]
> Once Yu-Ju removes the hold, we will proceed (let;s see a few green run as mentioned)

We have run into some issues with node crashing in Peter's fork running tests with the 1809 image. We are going to verify whether the new image was the cause before merging this PR.

**Comment by pjh** [comment]
Rebased with no merge conflicts.

**Pull request merged**
